ecs_deployer/conf.py

```diff
@@ -5,6 +5,9 @@
 VAULT_TOKEN = os.getenv('VAULT_TOKEN', None)
 VAULT_PATH = os.getenv('VAULT_PATH', None)
 
+AWS_SECRET_NAME = os.getenv('AWS_SECRET_NAME', None)
+AWS_REGION_NAME = os.getenv('AWS_REGION_NAME', None)
```

Comment:
The default value for `getenv` is None so there's no need for this second arg

Reply:
These lines will be fixed in another PR where we will _beautify_ the code removing these unnecessary Nones and add more typing to document better the methods